packages/core-ethereum/.gitignore

```diff
@@ -1,21 +1,14 @@
 # Exclude node.js stuff
-**/node_modules
-
-# Exclude keys
-.env
```

Comment:
This could very well be used by individuals locally.

Reply:
Yes, that's true. But we don't use the package `dotenv` in `hopr-core-ethereum`. This stem from the time when there was no `hopr-ethereum` and we did deployments out of `hopr-core-ethereum`